clio-server/src/main/scala/org/broadinstitute/clio/server/service/PersistenceService.scala

```diff
@@ -21,11 +23,13 @@ import java.util.UUID
   * Updates are written to storage to serve as the system of record for Clio,
   * allowing for update playback during disaster recovery.
   */
-class PersistenceService private (persistenceDAO: PersistenceDAO,
-                                  searchDAO: SearchDAO) {
+class PersistenceService private (
+  persistenceDAO: PersistenceDAO,
+  searchDAO: SearchDAO
```

Comment:
It seems like `SearchDAO` is not the right name for this type, since it supports operations other than search. Not sure what a better name would be, something like `MetadataStorageDAO` seems kind of wordy.

Reply:
If we were using it as a search API, that would make sense. But we're using it as a metadata storage/retrieval API.